pkg/cli/update.go

```diff
@@ -17,6 +17,15 @@ func NewUpdate(c CommandContext) *cobra.Command {
 		SilenceUsage:      true,
 		Short:             "Update a deployed app",
```

Comment:
Nit: Should this be "Update a deployed Acorn" then as well to align with the naming below? :thinking: